### PR TITLE
fix Makefile to pass instance_id into commands as env variable

### PR DESCRIPTION
**Pull request by marusinm**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 TEST_IMAGE := docker.io/zaitra/sentinel-downloader-tests
 TEST_TARGET := ./tests/
+INSTANCE_ID=${SD_SENTINEL_INSTANCE_ID}
 
 build: files/install-deps.yaml
 	docker build --rm -t $(IMAGE) .
 
 run-in-container: build
-	docker run -it --rm -v /tmp/images:/tmp/images $(IMAGE) bash -c "sentinel-downloader download -c /src/.sd.yaml"
+	docker run -it --rm -v /tmp/images:/tmp/images -e SD_SENTINEL_INSTANCE_ID=$(INSTANCE_ID) $(IMAGE) bash -c "sentinel-downloader download -c /src/.sd.yaml"
```
Comment (rpitonak) [inline Makefile]: I would like to see that everything in docker run command is configurable with default options. So you can run:
```
make run-in-container INSTANCE_ID=xyz LOCAL_IMAGE_DIR=/path/to/images CONFIG_FILE=/path/to/config
```
Defaults could be:
```
LOCAL_IMAGE_DIR=/tmp/images
CONFIG_FILE=$(CURRDIR)/.sd.yaml
```

Take this as a suggested workflow, I am open to discussion.

File: README.md [hunk 2]
```diff
@@ -73,9 +73,17 @@ $ make build  # builds docker image
...
-$ docker run -it --rm -v <PATH_TO_IMAGES>:/tmp/images -e SD_SENTINEL_INSTANCE_ID=<INSTANCE_ID> sd-image bash -c "sentinel-downloader download -c /src/.sd.yaml"
+$ docker run -it --rm -v <PATH_TO_IMAGES>:/tmp/images -e SD_SENTINEL_INSTANCE_ID=<INSTANCE_ID> zaitra/sentinel-downloader:dev bash -c "sentinel-downloader download -c /src/.sd.yaml"
 ```
+or use Makefile
+```bash
+$ echo "export SD_SENTINEL_INSTANCE_ID=<INSTANCE_ID>" >> .env
+$ source .env
+$ make build run-in-container
+```
+>Default path for downloaded images is `/tmp/images`. You can edit Dockerfile to change it.
```
[the remark this answers is not in the document]
Comment (marusinm) [reply README.md]: It is actually in Makefile in this case

File: README.md [hunk 3]
```diff
@@ -73,9 +73,18 @@ $ make build  # builds docker image
...
 ## Usage
 
+Run docker command
 ```bash
-$ docker run -it --rm -v <PATH_TO_IMAGES>:/tmp/images -e SD_SENTINEL_INSTANCE_ID=<INSTANCE_ID> sd-image bash -c "sentinel-downloader download -c /src/.sd.yaml"
+$ docker run -it --rm -v <PATH_TO_IMAGES>:/tmp/images -e SD_SENTINEL_INSTANCE_ID=<INSTANCE_ID> zaitra/sentinel-downloader:dev bash -c "sentinel-downloader download -c /src/.sd.yaml"
 ```
+or use Makefile
+```bash
+$ make run-in-container INSTANCE_ID=<INSTANCE_ID> LOCAL_IMAGE_DIR=<PATH_TO_DOWNLOADS_FILE> CONFIG_FILE=<PATH_TO_CONFIG_FILE>
```
Comment (rpitonak) [inline README.md]: `PATH_TO_DOWNLOADS_FILE` name does not make sense

File: README.md [hunk 4]
```diff
@@ -73,9 +73,18 @@ $ make build  # builds docker image
...
+```
+>Defaults are:  
+<PATH_TO_DOWNLOADS_FILE> = /tmp/images
```
Comment (rpitonak) [inline README.md]: same as above

File: Makefile [hunk 5]
```diff
@@ -1,12 +1,15 @@
 IMAGE := docker.io/zaitra/sentinel-downloader:dev
 TEST_IMAGE := docker.io/zaitra/sentinel-downloader-tests
 TEST_TARGET := ./tests/
+INSTANCE_ID := ${SD_SENTINEL_INSTANCE_ID}
+LOCAL_IMAGE_DIR := /tmp/images
+CONFIG_FILE := $(PWD)/.sd.yaml
```
Comment (rpitonak) [inline Makefile]: please use `$(CURDIR)` since `$(PWD)` causes problems on some linux distros.

**Pull request merged**